docs/rpc/bnb-rpc-documentation/abciQuerry.md

```diff
@@ -0,0 +1,46 @@
+# abciQuery
+
+### How to use it
+
+```typescript
+/// Importing Tatum SDK for Beacon Chain
+import { TatumSDK, Network, Bnb } from '@tatumio/tatum';
+
+// Initializing SDK for Beacon Chain network
+const tatum = await TatumSDK.init<Bnb>({ network: Network.BNB });
+
+// Defining query parameters for the 'abciQuery' method
+const queryParams = {
+  path: 'YOUR_PATH_HERE',   
```

Comment:
why once with '' and then without ''? :-D

Reply:
fixed